search-app/src/cmr/search/services/content_service.clj

```diff
@@ -0,0 +1,153 @@
+(ns cmr.search.services.content-service
+  "Provides service functions for generating static content."
+  (:require
+   [clojure.string :as string]
+   [cmr.common-app.cache.consistent-cache :as consistent-cache]
+   [cmr.common-app.cache.cubby-cache :as cubby-cache]
+   [cmr.common.cache :as cache]
+   [cmr.common.cache.fallback-cache :as fallback-cache]
+   [cmr.common.cache.single-thread-lookup-cache :as stl-cache]
+   [cmr.common.config :refer [defconfig]]
+   [cmr.common.jobs :refer [defjob default-job-start-delay]]
+   [cmr.common.log :refer :all]
+   [cmr.common.util :as util]
+   [cmr.search.site.pages :as pages]
+   [cmr.search.site.static :as static]
+   [cmr.search.site.util :as site-utils]
+   [cmr.transmit.metadata-db :as mdb]))
+
+(def cache-key
+  "The key used to store the humanizer report cache in the system cache map."
+  :static-content-cache)
+
+(defconfig job-delay
```

Comment:
`defconfig` doesn't take the namespace into account for configuration parameters. So this configuration would be controlled by an environment variable CMR_JOB_DELAY. That's too generic, we would want these configs to be more like CMR_STATIC_CONTENT_JOB_DELAY for example.

Reply:
Doh, forgot about that -- thanks.

Fixed.